### ISSUE TITLE
SD card not working?

### ISSUE THREAD

**Issue by wurzelmuschel** [bhall66/morse-tutor#1]
Hi,

if I insert an SD card into the Morse Tutor and click on "Receive -> SD Card", the file selector menu shows only gibberish. I formatted the SD card with a FAT file system and added a file with only five characters and a .txt extension, so there should not be any problem.

Interestingly enough it does not make a difference whether I insert an SD card into the card reader or not, the menu always shows a lot of "random noise" filenames, so I guess there is something broken with the SD Card library from the STM32 Arduino core.

I am using the latest Arduino IDE (1.8.10) with up-to-date libraries. Can someone verify this strange behavior?

Thanks,
Christoph

**Comment by bhall66** [comment]
Christoph,
Have you tried using one of the text files I supply with the code?   

Bruce
w8bh.net

**Comment by wurzelmuschel** [comment]
Bruce,

I just downloaded the files from the git repository onto a newly formatted SD card, and (of course) this time it worked. I don't know what exactly happened the last time I tried it, but two devices built by two OMs were not able to play back the files I downloaded, while they were readable on my PC.

Anyway, thanks again and sorry for bothering you. All is well that ends well ;-)

Cheers,
Christoph
